<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemperatureWindow</class>
 <widget class="QDialog" name="TemperatureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Temperature</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>313</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_setpoint">
      <property name="text">
       <string>Setpoint (°C): </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_measure">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_switch_position">
      <property name="text">
       <string>Switch: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_measure">
      <property name="text">
       <string>Measure (°C): </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="switch_radioButton_power">
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="switch_radioButton_unpower">
        <property name="text">
         <string>Unpower</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_refresh">
      <property name="text">
       <string>Refresh ctrl (s)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBox_refresh">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Period at which the system will check &lt;/p&gt;&lt;p&gt;if measured temperature is within setpoint range.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="maximum">
       <number>1800</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_setpoint">
        <property name="decimals">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_plus_minus">
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_hysteresis">
        <property name="toolTip">
         <string>Hysteresis</string>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_pump">
   <property name="text">
    <string>Syringe pump</string>
   </property>
  </action>
  <action name="action_flows">
   <property name="text">
    <string>Flow controllers</string>
   </property>
  </action>
  <action name="action_pressure">
   <property name="text">
    <string>Pressure controller</string>
   </property>
  </action>
  <action name="action_valve2x3">
   <property name="text">
    <string>Double 3-ways valve</string>
   </property>
  </action>
  <action name="action_electrovalves">
   <property name="text">
    <string>Electro-valves</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
